### PR TITLE
added local-edit button

### PR DESCRIPTION
**Pull request by Shahar-Y**
Signed-off-by: shahar-y <shya95@gmail.com>

### CODE REVIEW

File: src/i18n/he.json [hunk 1]
```diff
@@ -63,7 +63,8 @@
     "CopyLink": "העתקת הקישור",
     "FileInfo": "פרטי הקובץ",
     "FolderInfo": "פרטי התיקייה",
-    "EditOnline": "ערוך בזמן אמת",
+    "EditOnline": "עריכת online",
```
Comment (yoavrozner) [inline src/i18n/he.json:66]: Did you consult with Liron Ezra? Because she usually prefers Hebrew text

File: src/styles/global.css [hunk 2]
```diff
@@ -132,6 +132,16 @@ a {
   font-size: 50px;
 }
 
+.adapt-color-white {
```
Comment (yoavrozner) [inline src/styles/global.css:135]: These class are scoped please write them accordingly in the scoped section of the vue component.

File: src/i18n/he.json (hunk 1 above)
Comment (Shahar-Y) [reply src/i18n/he.json:66]: Yes, this was done with her assistance

File: src/styles/global.css (hunk 2 above)
Comment (Shahar-Y) [reply src/styles/global.css:135]: done

**Pull request merged**